.github/workflows/publish-storybook.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: main
+          fetch-depth: 0
```

Comment:
This is to try and fix the deployment issue we're seeing from the Storybook 7 build that was merged: https://github.com/department-of-veterans-affairs/component-library/actions/runs/9470553487/job/26091744901